cmd/viam-agent/main.go

```diff
@@ -211,7 +211,8 @@ func setupExitSignalHandling() (context.Context, context.CancelFunc) {
 			case syscall.SIGABRT:
 				fallthrough
 			case syscall.SIGTERM:
-				globalLogger.Info("exiting")
+				globalLogger.Infof("Signal %s was received. %s will now exit to be restarted by service manager",
```

Comment:
This change, and the addition and usage of `reason` for the `Exit` method in `manager.go` are for https://viam.atlassian.net/browse/RSDK-11266. Hopefully these messages will be useful for debugging.